src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -295,6 +295,84 @@ protected Object decode(
                                     value = buf.readByte();
                                     position.set(Position.PREFIX_TEMP + (j + 2 - 45), (value != 0x80) ? value : null);
                                     break;
+                                case 53:
+                                    value = buf.readUnsignedShortLE();
+                                    if (value == 0x7FFF) {
+                                        position.set(Position.KEY_FUEL_LEVEL + "Level", (Boolean) null);
+                                        position.set(Position.KEY_FUEL_LEVEL, (Boolean) null);
+                                    } else if (BitUtil.check(value, 7)) {
+                                        position.set(Position.KEY_FUEL_LEVEL + "Level", BitUtil.to(value, 6));
```

Comment:
Also what's `Position.KEY_FUEL_LEVEL + "Level"`?

Reply:
In the CAN bus, fuel data can be transmitted in liters or percentages.  For transmission to the server, we use one field.  If the most significant bit is 1, then the remaining bits are the fuel percentage value.  Otherwise, the remaining bits are the fuel value in liters * 0.1.  
Therefore, two separate parameters should be displayed for the user (as battery voltage - in volts and in percent).

Reply:
You should just call it "fuelLevel".

Reply:
ok, got it